assisted-events-scrape/workers/cluster_events_worker.py

```diff
@@ -93,3 +102,20 @@ def _drain_queue(self):
                 break
             if work_item is not None:
                 work_item.future.cancel()
+
+    def _store_normalized_events(self, component_versions, cluster, event_list):
+        try:
+            self._es_store.store_changes(
+                index=COMPONENT_VERSIONS_EVENTS_INDEX,
+                documents=[component_versions],
+                id_fn=get_dict_hash)
+            self._es_store.store_changes(
+                index=CLUSTER_EVENTS_INDEX,
+                documents=[cluster],
+                id_fn=get_dict_hash)
```

Comment:
For now we'll use this function without filtering, but I would like to see in staging what is the behaviour with realistic data and probably filter out some fields.
In any case if anybody knows already problematic fields, we shall add them in this very PR